src/v/redpanda/admin_server.cc

```diff
@@ -4422,12 +4422,7 @@ admin_server::unsafe_reset_metadata(
         throw ss::httpd::bad_request_exception("Empty request content");
     }
 
-    ss::sstring content = request->content;
-
-    iobuf buf;
-    buf.append(request->content.data(), request->content.size());
-
-    content = {};
+    ss::sstring content = std::move(request->content);
```

Comment:
fyi:

```
    sstring content; // server-side deprecated: use content_stream instead
    /*
     * The handler should read the contents of this stream till reaching eof (i.e., the end of this request's content). Failing to do so
     * will force the server to close this connection, and the client will not be able to reuse this connection for the next request.
     * The stream should not be closed by the handler, the server will close it for the handler.
     * */
    input_stream<char>* content_stream;
```

not that we need to deal with that here, but just flagging for future.

Reply:
>httpd::http_server::set_content_streaming(true)

oh cool. i was wondering how they made both apis work at the same time!